client/web/emberclear/app/services/messages/factory.ts

```diff
@@ -64,7 +64,7 @@ export default class MessageFactory extends Service {
     return this.build({
       type: TYPE.CHANNEL_VOTE,
       to: to.id,
-      metadata: buildVote(vote),
+      metadata: { [METADATA_VALUE.VOTE]: buildVote(vote) },
```

Comment:
what is the type of buildVote? was it not Record-like?

Reply:
buildVote returns a StandardMessage (an index type) and not record like